src/Services/Basket/Basket.API/Program.cs

```diff
@@ -1,28 +1,22 @@
-﻿using Basket.API.Infrastructure.Middlewares;
-using Microsoft.AspNetCore;
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Server.Kestrel.Core;
-using Microsoft.eShopOnContainers.Services.Basket.API;
-using Microsoft.Extensions.Configuration;
-using Azure.Identity;
-using Serilog;
-using System;
-using System.IO;
-using System.Net;
-using Azure.Core;
-
-var configuration = GetConfiguration();
-
+﻿var configuration = GetConfiguration();
```

Comment:
I'm a bit confused as to why we're adding so many new things to the _Program.cs_ file here? I'm not sure I like this, it adds a lot of complexity and I would much rather see these aspects encapsulated into extension methods or in separate files.

Reply:
As discussed with @nishanil, we are keeping the `Startup.cs` file to keep encapsulated methods along with the `Program.cs`. This change is invalid now. I have reverted back those previous commits.